### ISSUE TITLE
Unexpected cursor moving with "gg" combinated map

### ISSUE THREAD

**Issue by supermomonga** [vscode-neovim/vscode-neovim#904]
https://user-images.githubusercontent.com/377137/168517090-e42f5b41-2f11-4240-b2ab-25893c0ed7e6.mp4

## How to reproduce

```
aA1
bB2
cC3
dD4
eE5
```

1. Create a buffer using above text
2. Define mapping. `nnoremap a 3gg0l`
3. Move cursor to 2nd or 4th line (e.g. `2gg` / `4gg`)
4. Press `a`

## Expected behavior

Cursor moves to 2nd char of 3rd line (character "C")

## Actual behavior

- Cursor on 2nd line moves to 4th line
- Cursor on 4th line moves to 2nd line

If the cursor is on other lines, it moves to correct position (to "C")

## Environment

- Windows 11
- neovim v0.7.0
- VSCode Neovim v0.0.84

```
NVIM v0.7.0
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```

**Comment by theol0403** [comment]
Can reproduce on linux. Very strange, no idea why this is happening.

**Comment by zbw8388** [comment]
I think this is fixed by #775.
